### PR TITLE
Landing style

### PR DESCRIPTION
**Pull request by Shazeen15**

### CODE REVIEW

**Comment by Shazeen15** [comment]
i have some stuff from another local branch, that was before your comment to only create branches from the main

File: app/javascript/components/components/LandingPage.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-        <h2>Travel More</h2>
-        <p>
-          One place for all your trip planning{" "}
-          <span>a new way to share your travels</span>
-        </p>
-        <p>Want to travel to the same place again and loved your trip</p>
-        <p>Do it again with your saved trips</p>
+      <div className="landing-container">
+        {/* <video src="../../../assets/videos/video1.mp4" autoPlay loop muted /> */}
+        <div className="landing-container-2">
```
Comment (minachuong) [inline app/javascript/components/components/LandingPage.js:9]: looks like there are two divs with very similar class names `landing-container`. Try to specific the different intentions of the classes other wise it's a symptom of some possible need for refactoring the markup

File: app/javascript/components/components/__test__/Dash.test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          trip_id: 1,
+        },
+      ],
+    },
+  ];
+  it("displays welcome message to user", () => {
+    const dash = shallow(<Dash trips={mocktrips} user={mockuser} />);
+
+    const dashWelcomeMess = dash.find("h1");
+
```
Comment (minachuong) [inline app/javascript/components/components/__test__/Dash.test.js:39]: awesome. Definitely on the right path with front end tests. More assertions on other things a user should expect to see on this page would help make your tests more useful

File: app/javascript/components/components/__test__/Footer.test.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import Adapter from "enzyme-adapter-react-16";
+import Footer from "../Footer";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("When footer renders", () => {
+  it("displays a link to home '/'", () => {
+    const footer = shallow(<Footer />);
+    const footerlinks = footer.find("NavLink");
+    expect(footerlinks.length).toEqual(2);
```
Comment (minachuong) [inline app/javascript/components/components/__test__/Footer.test.js:12]: Are there different texts for each link? If so, that'd be a good test to add

**Review by minachuong — APPROVED** [review]
👍 Thanks for making the suggested changes!

**Pull request merged**